synapse/lib/cell.py

```diff
@@ -89,6 +89,23 @@ def getCellIden(self):
 
         raise s_exc.AuthDeny(mesg='Caller must own task or be admin.', task=iden, user=str(self.user))
 
+    async def hivels(self, path=None):
+        if path is None:
+            path = ()
+        items = self.cell.hive.dir(path)
+        if items is None:
+            return None
+        return [item[0] for item in items]
+
+    async def hivegetkey(self, path):
```

Comment:
getHiveDir (or something) / getHiveKey / setHiveKey / popHiveKey ( since these are published / remote APIs ) is more consistent with out other multi-word API names. :D

Reply:
ok